### ISSUE TITLE
Table.AddRow (string value) throws an exception if one row is too long.

### ISSUE THREAD

**Issue by aijkl** [spectreconsole/spectre.console#528]
**Information**
 - OS: Windows10
 - Version: latest
 - Terminal: VisualStudio DebugConsole

**Describe the bug**
Table.AddRow (string value) throws an exception if one row is too long.
System.ArgumentOutOfRangeException:'Index and length must refer to a location within the string. Arg_ParamName_Name'

**To Reproduce**

```
Table table = new Table();
            table.AddColumn("Japanese Text");
            table.AddRow(@"ここ良かった！街、クラブ、個室とかなりよく作り込まれてる。ゲームコーナーやシューティングレンジもあって、UdonChipsも使われてた。銃が気持ちいい。全体的に音周りの調整が凄いので参考にさせてもらいたいワールド。
                           #VRChat_world紹介
                           The AfterDark Arcade https://t.co/wjrc8HJc1W");
            AnsiConsole.Render(table);
```

**Expected behavior**
Surrogate characters were not included, which is most likely a problem with character length.

**Screenshots**
![image](https://user-images.githubusercontent.com/51302983/132082227-f9062a23-fe22-48e6-ba98-546640d05408.png)

I love this library, thank you always. Hope it will be fixed

**Comment by patriksvensson** [comment]
I suspect this has been resolved in #501.
Could you reference version `0.41.1-preview.0.12` and see if you're still seeing this error?

**Comment by aijkl** [comment]
![image](https://user-images.githubusercontent.com/51302983/132112323-7c5d3651-d01c-4b47-adb7-0a30e35055ce.png)
I was able to confirm that it was fixed in ```0.41.1-preview.0.12.```
thank you
